src/setup.py

```diff
@@ -24,6 +21,45 @@
 # 5 - Success message (or failed)
 # 6 - ask the user delete the folder and exit
 
+# Verifying if all required files are included in /dist/ folder
+try:
+
+    print("Verifying required files...")
+
+    source = os.path.join(os.path.abspath(os.curdir), "discordRPC\\dist")
+
+    # If any files are needed within the /dist/ folder, add them here
+    requiredFiles = ("checkupdate.exe", "hoi4Presence.exe", "version.json")
+    filesNotFound = []
+
+    for file in requiredFiles:
```

Comment:
Also add a verification for runRPC.bat, it is in /batch folder, you can do it using an if condition or refactoring your logic to check for the folder and the file

Reply:
I don't know if it is safe to move runRPC.bat to dist folder since it can break and reach some unexpected code. You can analyze it if it's safe.

See `batchPath` variable.

Reply:
> I don't know if it is safe to move runRPC.bat to dist folder since it can break and reach some unexpected code. You can analyze it if it's safe.

I did some testing and I think its very possible to do that.

I cannot fully test it since I don't own the game but, I made a mock .exe with a txt file and I was able to get runRPC.bat to run by changing line 5 from:

```batch
start "hoi4.exe" "./hoi4.exe"
```

To:
```batch
start "hoi4.exe" "%documentsPath%\hoi4.exe"
```

Reply:
> Also add a verification for runRPC.bat, it is in /batch folder, you can do it using an if condition or refactoring your logic to check for the folder and the file

Instead of including a if statement in a for loop I added a bool variable that checks for runRPC.bat. Detailed notes are in the new [commit](https://github.com/ThiaudioTT/hoi4-presence/pull/24/commits/c2da7dda4d51c425309fe06ba5848b13faff85be)